### PR TITLE
Remove 'All rights reserved' from copyright headers

### LINKED ISSUE #50: Remove 'All rights reserved' from the copyright headers
Since July 2019 Microsoft no longer recommend the term All Rights Reserved to be in MIT license copyright headers.

PowerShell repo has already made these changes:
https://github.com/PowerShell/PowerShell/pull/12190

We should update the headers to match the following :

> // Copyright (c) Microsoft Corporation.
> // Licensed under the MIT License.

See: https://docs.opensource.microsoft.com/content/releasing/copyright-headers.html
(link may require Microsoft credentials)

### PR DESCRIPTION
**Pull request by xtqqczze**
Fix #50

### CODE REVIEW

**Comment by xtqqczze** [comment]
@adityapatwardhan can we merge this?

**Comment by xtqqczze** [comment]
@iSazonov can you help merge?

**Comment by iSazonov** [comment]
@xtqqczze No, I am not a maintainer in the repository.

File: src/powershell-native/nativemsh/pwrshplugin/pwrshpluginResources.rc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation.
 //
```
Comment (adityapatwardhan) [inline src/powershell-native/nativemsh/pwrshplugin/pwrshpluginResources.rc:2]:
```suggestion
// Licensed under the MIT License.

```

File: src/powershell-native/nativemsh/pwrshexe/version.rc [hunk 2]
```diff
@@ -1,5 +1,5 @@
 //
-//    Copyright (c) Microsoft Corporation. All rights reserved.
+//    Copyright (c) Microsoft Corporation.
 //
```
Comment (adityapatwardhan) [inline src/powershell-native/nativemsh/pwrshexe/version.rc:3]:
```suggestion
//     Licensed under the MIT License.

```

**Comment by xtqqczze** [comment]
@adityapatwardhan can you approve?

File: src/powershell-native/nativemsh/pwrshexe/version.rc [hunk 3]
```diff
@@ -1,6 +1,7 @@
...
 //    Copyright (c) Microsoft Corporation.
-//
+//     Licensed under the MIT License.
```
Comment (adityapatwardhan) [inline src/powershell-native/nativemsh/pwrshexe/version.rc:3]: Please fix indentation

**Comment by xtqqczze** [comment]
@adityapatwardhan I've set a consistent header for `.rc` files and fixed indentation:
```
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
```

**Comment by adityapatwardhan** [comment]
@xtqqczze Thank you for your contribution!

**Pull request merged**